src/library/librarycontrol.cpp

```diff
@@ -1109,24 +1126,17 @@ void LibraryControl::slotDecrementFontSize(double v) {
     }
 }
 
-void LibraryControl::slotTrackColorPrev(double v) {
-    if (!m_pLibraryWidget || v <= 0) {
+void LibraryControl::slotTrackColorSelector(int steps) {
+    if (!m_pLibraryWidget || steps == 0) {
         return;
     }
 
-    WTrackTableView* pTrackTableView = m_pLibraryWidget->getCurrentTrackTableView();
-    if (pTrackTableView) {
-        pTrackTableView->assignPreviousTrackColor();
-    }
-}
-
-void LibraryControl::slotTrackColorNext(double v) {
-    if (!m_pLibraryWidget || v <= 0) {
+    LibraryView* pActiveView = m_pLibraryWidget->getCurrentTrackTableView();
```

Comment:
Please change to WTrackTableView
```suggestion
    WTrackTableView* pTrackTableView = m_pLibraryWidget->getCurrentTrackTableView();
```
and use `pTrackTableView` below.